build/configure-ct

```diff
@@ -39,5 +39,5 @@ then
       ;;
   esac
 
-  ./configure GMOCK_DIR=${GMOCK} ${CONFIGURE_FLAGS} PROTOC=${INSTALL_DIR}/bin/protoc PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig" CPPFLAGS="-I${INSTALL_DIR}/include" CXXFLAGS="${CXXFLAGS} ${EXTRA_CXXFLAGS}" LDFLAGS="-L${INSTALL_DIR}/lib $EXTRA_LDFLAGS" LIBS="${EXTRA_LIBS}" INSTALL_DIR=${INSTALL_DIR} || tail -1000 config.log
+  ./configure GMOCK_DIR=${GMOCK} ${CONFIGURE_FLAGS} PROTOC=${INSTALL_DIR}/bin/protoc PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig" CPPFLAGS="-I${INSTALL_DIR}/include" CXXFLAGS="${CXXFLAGS} ${EXTRA_CXXFLAGS}" LDFLAGS="-L${INSTALL_DIR}/lib $EXTRA_LDFLAGS" LIBS="${EXTRA_LIBS}" evhtp_CFLAGS="-I${INSTALL_DIR}/include/evhtp" evhtp_LIBS="-levhtp" INSTALL_DIR=${INSTALL_DIR} || tail -1000 config.log
```

Comment:
Ok, I looked at this closer...

https://github.com/ellzey/libevhtp/commit/7065156a31a37e3b69f1d04d873ac34af87acc3c

In 1.2.10 (which we're still using, as of this PR), the headers are in `${INSTALL_DIR}/include`, they've changed the location in 1.2.11.

What I would say is this: right now, don't preemptively put the 1.2.11 location, but make it what needs to compile (which might be even `evhtp_CFLAGS=""`!), and when we do upgrade to 1.2.11, we'll switch away from specifying `evhtp_CFLAGS` and `evhtp_LIBS` and let the `configure` script get them from `pkg-config`, which will then "just work".